### ISSUE TITLE
Test failure on v46

### ISSUE THREAD

**Issue by jesperfj** [jesperfj/force-rest-api#80]
When going from v45 to v46, com.force.api.BasicCRUDTest.basicCRUDTest fails. v46 returns code 200 where v45 returns expected code (201 or 204). What changed in Salesforce REST API between 45 and 46?

**Comment by jesperfj** [comment]
To reproduce, set default version in `ApiVersion` to 46:

    DEFAULT_VERSION ("v46.0");

Then run the test with debug loggging:

    mvn test -Dtest=com.force.api.BasicCRUDTest -Dorg.slf4j.simpleLogger.defaultLogLevel=debug

```
Running com.force.api.BasicCRUDTest
Setting test config from environment
[main] DEBUG com.force.api.ForceApi - Code: 200
[main] DEBUG com.force.api.ForceApi - Message: null
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.761 sec <<< FAILURE!
basicCRUDTest(com.force.api.BasicCRUDTest)  Time elapsed: 4.718 sec  <<< ERROR!
java.lang.RuntimeException
	at com.force.api.ForceApi.createOrUpdateSObject(ForceApi.java:275)
	at com.force.api.BasicCRUDTest.basicCRUDTest(BasicCRUDTest.java:64)
```

This behavior might be correct. But the error does not happen with API version 45 or earlier.

**Comment by jesperfj** [comment]
This is fixed in v0.0.43
